README.md

```diff
@@ -1,45 +1,3 @@
-# @blackbaud/skyux-lib-help
+# @blackbaud/skyux-lib-help Workspace
 
-[![npm](https://img.shields.io/npm/v/@blackbaud/skyux-lib-help.svg)](https://www.npmjs.com/package/@blackbaud/skyux-lib-help)
-
-Provides a SKYUX library for interacting with the Help Widget. Wraps much of the functionality of [@blackbaud/help-client] in SKYUX-compatible fashion. This module is a dependency of [@skyux-sdk/builder].
-
-## SKYUX usage
-
-Enable the help widget by providing a configuration in `skyuxconfig.json`. `@skyux-sdk/builder` will initialize the widget based on this config.
-
-```json
-{
-  "help": {
-    "helpMode": "menu"
-  }
-}
-```
-See [widget-config.ts] for more configuration properties.
-
-Use the widget by injecting the `HelpWidgetService` into your desired directive/service.
-
-```typescript
-@Component({
-  selector: 'my-comp',
-  template: `
-    <button (click)="open()">Help</button>
-  `
-})
-export class MyComponent {
-  public constructor(private helpSvc: HelpWidgetService) {
-  }
-
-  public open(): void {
-    this.helpSvc.openToHelpKey('bb-custom-fields.html');
-  }
-}
-```
-
-# Menu vs legacy mode
-
-See [@blackbaud/help-client] for more information on the `helpMode` property. `menu` is the recommended mode going forward.
-
-[@skyux-sdk/builder]: https://github.com/blackbaud/skyux-sdk-builder
-[@blackbaud/help-client]: https://github.com/blackbaud/help-client
-[widget-config.ts]: src/widget-config.ts
+For information about `@blackbaud/skyux-lib-help` please visit the library's README file, located at `projects/help/README.md`.
```

Comment:
this seems a bit odd since the showcase does not have a README. there's only one readme here. might as well let it be at the top as opposed making folks dive deeper. granted, not many ppl look at this readme so 🤷 . but in terms of a convention for BB libs, i'd recommend j/ keeping the readme at the root since the showcase is likely to never have a README. (and even if it does, consumer's likely aren't coming to the project to look at the showcase's README.)

at bare minimum, make this a link to the inner readme to aid the consumer coming here looking for it.